app/controllers/cohorts/columns_controller.rb

```diff
@@ -28,9 +28,35 @@ def update
         end
       end
       @cohort.update(column_state: columns)
+      if params.include? :translation
+        translate_columns(params[:translation])
+      end
       respond_with(@cohort, location: cohort_path(@cohort))
     end
 
+    def translate_columns(translations)
+      return unless can_edit_translations?
```

Comment:
Unless we need to use this permission, it's probably more useful to  use `can_edit_cohorts` here.